components/admin_console/schema_admin_settings.jsx

```diff
@@ -812,10 +816,6 @@ export default class SchemaAdminSettings extends React.Component {
                 if (callback) {
                     callback();
                 }
-
-                if (this.handleSaved) {
```

Comment:
Why remove this?

Reply:
The reason I removed this was I couldn't find any corresponding any handleSaved function in the file and thought maybe this was added by accident so I decided to remove it. I just wasn't quite sure why we had it and where it's corresponding function call is. I'll add it back in but would just like to know where the corresponding handleSaved function is. Thanks!

Reply:
I've added this back in. I'm just not sure why we have it there or what the purpose is? Thanks!